samples/boards/mec15xxevb_assy6853/power_management/prj.conf

```diff
@@ -5,7 +5,7 @@ CONFIG_LOG=y
 CONFIG_LOG_PROCESS_THREAD_SLEEP_MS=2100
 
 # Enable kernel debug
-CONFIG_KERNEL_DEBUG=y
+CONFIG_KERNEL_DEBUG=n
```

Comment:
Remove line 7 and 8 if this PR needs updating for other reasons.

Reply:
Removed